source/index.txt

```diff
@@ -22,16 +16,60 @@ The Node driver documentation is organized as follows:
   /api-documentation
   /faq
   /issues-and-help
+  /compatibility
 
-Compatibility
--------------
 
-MongoDB Compatibility
-~~~~~~~~~~~~~~~~~~~~~
+Introduction
+------------
+The official MongoDB Node.js driver allows Node.js applications to connect and perform queries with MongoDB.
+
+.. note::
+   
+   These docs are for version 3.5 of the MongoDB Node.js driver. If
+   you're looking for an older version of the MongoDB Node.js driver
+   docs, `see the GitHub pages
+   <https://mongodb.github.io/node-mongodb-native/>`_. For the main
+   MongoDB documentation, see the :manual:`MongoDB Manual </>`.
+
+Quick Start
+-----------
+Learn how to quickly get a Node.js application working with MongoDB Atlas through the `Quick Start </quick-start/>`_ section
+
+Fundamentals
+------------
+The `Fundamentals </fundamentals>`_ section defines the following core concepts involved with using the MongoDB
+Node.js driver: 
 
-.. include:: /includes/mongodb-compatibility-table-node.rst
+* CRUD: Learn how to read and write data to MongoDB using Node.js
+* Authentication: Learn how to log a user in through various community and enterprise mechanisms
+* Logs: Learn how MongoDB logs operations and how to work with them
+* Monitoring: Learn how to monitor server events
 
-Language Compatibility
-~~~~~~~~~~~~~~~~~~~~~~
+Usage Examples
+--------------
+Learn how to use common MongoDB Node.js driver methods through brief
+explanation and opinionated code snippets in the `Usage Examples </usage-examples/overview>`_ section.
+This section is best suited for new users to the driver who wants to
+implement common driver methods quickly. 
```

Comment:
I wonder if we should open with the persona instead of the description. Interested in everyone's thoughts here.

I think it would reduce the number of paragraphs that start with "Learn how..." and may help users more quickly identify the section is for them.

Reply:
"The `Usage Example <link>` section provides code snippets and explanation for the common methods, best suited for new users to the MongoDB Node.js driver who want to implement common driver methods"

Reply:
Yeah I like that! Maybe we could also mention that they're copyable/runnable examples?

Reply:
Fixed to:

"The Usage Example <link> section provides runnable code snippets and explanation for the common methods, best suited for new users to the MongoDB Node.js driver who want to implement common driver methods"

did include "runnable" just not "copyable" (we don't want to encourage copy-pasta users, but if they exists [and they do] they have the copy button)